pkg/core/helper_test.go

```diff
@@ -391,6 +391,14 @@ func TestCreateBasicChain(t *testing.T) {
 			require.NoError(t, writer.Err)
 		}
 	}
+
+	// Block for `submitblock` test. If you are planning to modify test chain from `testblocks.acc`,
+	// please, update param value of `submitblock positive` test.
+	b = bc.newBlock(newMinerTX())
+	buf := io.NewBufBinWriter()
+	b.EncodeBinary(buf.BinWriter)
+	require.NoError(t, buf.Err)
+	fmt.Println(hex.EncodeToString(buf.Bytes()))
```

Comment:
`t.Log` would be more appropriate here.

Reply:
Done. Also added one more test block without txs